src/instructlab/model/evaluate.py

```diff
@@ -405,6 +405,14 @@ def launch_server(
     default="",
     help="TLS client certificate password for model serving.",
 )
+@click.option(
+    "--device",
+    type=click.STRING,
+    default="",
+    cls=clickext.ConfigOption,
+    config_sections="mmlu",
+    help="PyTorch device (e.g. 'cpu' or 'cuda:0') for running models.",
+)
```

Comment:
We only support a view device types. Also we got reports that `cuda:0` does not work. Users have to set env vars to select AMD and Nvidia devices. I suggest that you replace `cuda:0` with `cuda` and use `type=click.Choice(["cpu", "cuda"])`. `cuda` also covers AMD ROCm. Let's not include `hpu` until we have tested the code with Intel Gaudi.

Reply:
Thank you! I'm going with cpu, cuda, and mps. Can add on as needed